tests/vcs/git/GitCommitBearTest.py

```diff
@@ -16,6 +16,79 @@
 from coalib.settings.Setting import Setting
 
 
+@generate_skip_decorator(GitCommitBear)
+class FakeGitHubMergeCommitBear(unittest.TestCase):
+
+    def run_uut(self, *args, **kwargs):
+        """
+        Runs the unit-under-test (via `self.uut.run()`) and collects the
+        messages of the yielded results as a list.
+
+        :param args:   Positional arguments to forward to the run function.
+        :param kwargs: Keyword arguments to forward to the run function.
+        :return:       A list of the message strings.
+        """
+        return list(result.message for result in self.uut.run(*args, **kwargs))
+
+    def setUp(self):
+        self.msg_queue = Queue()
+        self.section = Section('')
+        self.uut = GitCommitBear(None, self.section, self.msg_queue)
+
+        self._old_cwd = os.getcwd()
+        self.gitdir = mkdtemp()
+        os.chdir(self.gitdir)
+        GitCommitBearTest.run_git_command('init')
+        GitCommitBearTest.run_git_command(
+            'config', 'user.email coala@coala.io')
+        GitCommitBearTest.run_git_command('config', 'user.name coala')
+
+    def test_github_PR_merge_commit_offline(self):
```

Comment:
I think we can move this test into `GitCommitBearTest` class, since this test is an offline version of `test_github_pull_request_temporary_merge_commit`, and by that, more few code duplication is better